frontend/src/components/common/title-bar/TitleBarWithTabs.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ReactNode } from 'react'
+
+import defaultStyles from './TitleBar.module.scss'
+
+const HStackStyle = {
+  display: 'flex',
+  alignItems: 'center',
+}
+
+const subsequentElementStyle = { marginLeft: '4vw' }
+
+export const TitleBarWithTabs = ({
+  children,
+  title,
+  overrideStyles,
+}: {
+  children?: ReactNode
+  title: string
+  overrideStyles?: { readonly [key: string]: string }
+}) => {
+  const styles = overrideStyles || defaultStyles
+  return (
+    <div className={styles.titleBar}>
+      {!!children && Array.isArray(children) && children.length === 2 && (
```

Comment:
Could I ask what `!!` does?

Reply:
[Here you go](https://www.joshwcomeau.com/operator-lookup?match=double-not)

Reply:
Thank you!